### PR TITLE
Ndlano/issues#1520 upgrade notes

### PR DESCRIPTION
**Pull request by chrpeter**
Adds table that shows the different notes with status, names and timestamp including the note.

- [ ] Should be able to create new notes.
When a new note is created (the draft is saved), the following information should show:
- [ ] Correct timestamp should show.
- [ ] Correct name on the user that created the note
- [ ] Status when the note was created.
- [ ] The actual note.

### CODE REVIEW

**Comment by oyvinmar** [comment]
Merknad input should have some autosize functionality, not necessary to include in this PR but an issue should be created.

**Comment by chrpeter** [comment]
Dependent on https://github.com/NDLANO/draft-api/pull/93

File: src/containers/Form/FormAddNotes.jsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        name,
+      },
+    });
+  }
+
+  addNote() {
+    const { value } = this.props;
+    const newNotes = [].concat(value);
+    newNotes.push('');
+    this.onNotesChange(newNotes);
```
Comment (kodeebo) [inline src/containers/Form/FormAddNotes.jsx]: Penere å skrive `this.onNotesChange([...value, ''])`?

File: src/containers/Form/FormAddNotes.jsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  }
+
+  removeNote(e, index) {
+    const { value } = this.props;
+    e.preventDefault();
+    const newNotes = [].concat(value);
+    newNotes.splice(index, 1);
+    this.onNotesChange(newNotes);
```
Comment (kodeebo) [inline src/containers/Form/FormAddNotes.jsx]: `this.onNotesChange([ ...value.filter(_, i => i !== index]);` ?

File: src/containers/Form/components/FormNotes.jsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    };
+  }
+
+  async componentDidMount() {
+    const { notes } = this.props;
+    const userIds = notes.map(note => note.user);
+    const uniqueUserIds = Array.from(new Set(userIds)).join(',');
+    const users = await fetchAuthorized(
+      `/get_note_users?userIds=${uniqueUserIds}`,
+    ).then(resolveJsonOrRejectWithError);
```
Comment (kodeebo) [inline src/containers/Form/components/FormNotes.jsx]: Bruk await i stedet for then. Og kanskje flytt til api-fil slik som andre steder?

File: src/containers/LearningResourcePage/components/LearningResourceForm.jsx [hunk 4]
```diff
@@ -96,7 +93,7 @@ export const getInitialModel = (article = {}, language) => {
     language: language || article.language,
     articleType: 'standard',
     status: article.status || [],
-    notes: article.notes || [],
+    notes: [],
```
Comment (kodeebo) [inline src/containers/LearningResourcePage/components/LearningResourceForm.jsx]: Hvorfor? Hvor lastes eksisterende notes?

**Pull request merged**